tests/suites/main_test.function

```diff
@@ -221,8 +224,22 @@ int main( int argc, const char *argv[] )
                          ret );
         return( -1 );
     }
+
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+    {
```

Comment:
Why did you add the outer layer of curly braces?

Reply:
We must declare local variables at the beginning of their scope, otherwise the compilers complain.

Reply:
Hopefully we can soon relax that restriction when we move to using more of C99, but currently we do indeed compile with `-Wdeclaration-after-statement` until (a larger subset of) C99 is officially declared acceptable for us to use.

Reply:
Sorry, I asked because I was curious about why didn't you opt for having the declarations at the beginning of the function, therefore - omitting these braces. But I guess that's just a matter of style.